components/org.wso2.carbon.identity.oauth/pom.xml

```diff
@@ -324,6 +324,10 @@
             <groupId>org.wso2.carbon.identity.framework</groupId>
             <artifactId>org.wso2.carbon.identity.central.log.mgt</artifactId>
         </dependency>
+        <dependency>
```

Comment:
change the year range in license

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2196/commits/1f6b9ab57f76c33caa73487e50b8cab7a2c3a317